php/php-8.2.0-slim/patches

```diff
@@ -0,0 +1 @@
+../php-8.2.0/patches
```

Comment:
It turned out that using a symbolic link was the best way to solve this issue of sharing patches between flavors.
- Link the entire patch folder if we reuse all the patches.
- If a flavor only needs a few, just link only them.
- Etc.